### PR TITLE
feat: dict increment and remove

### PR DESCRIPTION
**Pull request by pgautier404**

### CODE REVIEW

File: src/Cache/_ScsDataClient.php [hunk 1]
```diff
@@ -401,6 +411,7 @@ public function dictionarySet(string $cacheName, string $dictionaryName, string
             validateFieldName($field);
             validateValueName($value);
             $ttlMillis = $this->ttlToMillis($ttlSeconds);
+            validateTtl($ttlMillis);
```
Comment (poppoerika) [inline src/Cache/_ScsDataClient.php:414]: Thank you!

File: src/Cache/_ScsDataClient.php [hunk 2]
```diff
@@ -542,4 +553,82 @@ public function dictionaryGetBatch(string $cacheName, string $dictionaryName, ar
...
+    }
+
+    public function dictionaryRemoveFields(string $cacheName, string $dictionaryName, array $fields): CacheDictionaryRemoveFieldsResponse
+    {
+        try {
+            validateCacheName($cacheName);
+            validateDictionaryName($dictionaryName);
+            foreach ($fields as $field) {
+                validateFieldName($field);
+            }
```
Comment (poppoerika) [inline src/Cache/_ScsDataClient.php:616]: Do we also want to validate `$fields` array is not an empty array?

**Review by poppoerika — COMMENTED** [review]
Thank you for updating all the conflicts...!!!

**Review by cprice404 — DISMISSED** [review]
🙈 i'm good when @poppoerika is

File: src/Cache/_ScsDataClient.php (hunk 2 above)
Comment (pgautier404) [reply src/Cache/_ScsDataClient.php]: Yes, we do! Thanks, good catch. Fixed in 6ea1091.

**Review by poppoerika — APPROVED** [review]
🎉

**Pull request merged**